src/environments/environment.prod.ts

```diff
@@ -1,3 +1,13 @@
 export const environment = {
-  production: true
+  production: true,
+  firebaseConfig :{
```

Comment:
looks like settings are same for prod & non-prod. have only env.ts file or else create two separate configuration for prod & non-prod